architecture/additional_concepts/admission_controllers.adoc

```diff
@@ -121,3 +121,10 @@ ifdef::openshift-dedicated[]
 - xref:../../admin_guide/osd_build_defaults_overrides.adoc#admin-guide-osd-build-defaults-overrides[Configuring Global Build Defaults and Overrides]
 endif::[]
 - xref:../../admin_guide/scheduler.adoc#controlling-pod-placement[Controlling Pod Placement]
+
+[[admission-controllers-and-containers]]
+== Admission Controllers and Containers
```

Comment:
@ahardin-rh I feel like this title is saying "Admission controllers and admission containers", but I don't think that's the meaning. Could it be something like "Using admission controllers in containers" or "Admission controllers that use containers"?